.github/actions/install-whitaker/tests/test_install_whitaker.py

```
@@ -0,0 +1,174 @@
+        )
+
+    env = {
+        **os.environ,
+        "PATH": f"{bin_dir}{os.pathsep}/usr/bin{os.pathsep}/bin",
+        "BINSTALL_AVAILABLE": str(binstall_available).lower(),
+        "CARGO_LOG": cargo_log.as_posix(),
+        "FAKE_BIN_DIR": bin_dir.as_posix(),
+        "INSTALLER_LOG": installer_log.as_posix(),
+        "WHITAKER_INSTALLER_VERSION": "0.2.6",
+    }
+    return subprocess.run(  # noqa: S603,TID251 - exercise the Bash fragment.
```

Ruff (0.15.21)

[warning] `subprocess.run` without explicit `check` argument

Add explicit `check=False`

(PLW1510)